keyboards/ciaanh/kanagawa/ansi/keymaps/default/keymap.c

```diff
@@ -0,0 +1,216 @@
+         XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,     RGB_HUI,           \
+         XXXXXXX, XXXXXXX, XXXXXXX, _______,         XXXXXXX,       XXXXXXX,              RGB_SPD, RGB_SPI, TO_BASE,     RGB_SAD, RGB_HUD, RGB_SAI   \
+    ),
+
+    [_TOP] = LAYOUT(
+         XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,  XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,             XXXXXXX,  \
+
+         XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,     XXXXXXX,  \
+         XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,     XXXXXXX,  \
+         XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,              XXXXXXX,  \
+         XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,     XXXXXXX,           \
+         XXXXXXX, XXXXXXX, XXXXXXX, _______,         XXXXXXX,       XXXXXXX,              XXXXXXX, XXXXXXX, XXXXXXX,     XXXXXXX, XXXXXXX, XXXXXXX   \
+    ),
+
+};
+
+/*****************************************************************************************************/
+
+#ifdef OLED_ENABLE
+    static void render_status(void) {
+        // 21 characters per line
+        // 16 cols / 4 rows
+
+        oled_write_P(PSTR("    Kanagawa rev 2.1   "), false);
+        //oled_write_ln("", false);
+
+        // Host Keyboard Layer Status
+        oled_write_P(PSTR("Layer: "), false);
+        switch (get_highest_layer(layer_state)) {
+            case _BASE:
+                oled_write_P(PSTR("_\n"), false);
+                break;
+            case _FUNC:
+                oled_write_P(PSTR("[Fn]\n"), false);
+                break;
+            case _CODE:
+                oled_write_P(PSTR("[Code]\n"), false);
+                break;
+            default:
+                oled_write_P(PSTR("Undefined\n"), false);
+        }
+
+        // Host Keyboard LED Status
+        led_t led_state = host_keyboard_led_state();
+        oled_write_ln(led_state.caps_lock ? "CAPLOCK" : "       ", false);
+        // oled_write_P(PSTR("         "), false);
+        
+        // oled_write_P(led_state.num_lock ? PSTR("NUM ") : PSTR("    "), false);
+        // oled_write_P(led_state.caps_lock ? PSTR("CAP ") : PSTR("    "), false);
+        // oled_write_P(led_state.scroll_lock ? PSTR("SCR ") : PSTR("    "), false);
+
+        oled_write_ln("", false);
+        // oled_write_ln("", false);
+        // oled_write_ln("", false);
+        // oled_write_ln("", false);
+        // oled_write_ln("", false);
+    }
+
+
+    bool oled_task_user(void) {
```

Comment:
the oled code here should be moved to `kanagawa.c` and use the oled_task_kb function.  Eg: 
```c
bool oled_task_kb(void) {
    if (!oled_task_user()) {
        return false;
    }
    // normal oled code

    return true;
}
```

This provides for a default behavior, without the need to specify it for each keymap (useful for the configurator, for instance).  This also reduces duplicate code.